docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/popover/popover.twig

```diff
@@ -0,0 +1,33 @@
+{#
+/**
+ * @file
+ * Popover component story.
+ *
+ * Variables:
+ * - theme: [string] Theme: light, dark.
+ * - trigger_event: [string] Trigger event: click, hover.
+ * - trigger_text: [string] Trigger text.
+ * - attributes: [string] Additional attributes.
+ * - modifier_class: [string] Additional classes.
+ */
+#}
+
+{% set trigger_event = trigger_event in ['click', 'hover'] ? trigger_event : 'click' %}
+{% set theme_class = 'ct-theme-%s'|format(theme|default('light')) %}
+{% set modifier_class = '%s %s'|format(theme_class, modifier_class|default('')) %}
+
+{% if content %}
+  <div class="ct-popover {{ modifier_class }}" data-popover {% if attributes is not empty %}{{ attributes|raw }}{% endif %}>
+    {% set trigger_attributes = 'data-popover-trigger data-popover-trigger-event="' ~ trigger_event ~ '"' %}
```

Comment:
@joshua-salsadigital 

this trigger should be moved into stories as it is not a part of popover. Trigger is any triggering element.